### ISSUE TITLE
How can I decode event timestamp ?

### ISSUE THREAD

**Issue by dimasmamot** [John-Lin/snort-socket#4]
I try to get the 
`msg.pkth.ts.tv_sec` and `msg.pkth.ts.tv_sec` but it's always at the wrong value,

and then I try to get the timestamp inside the 'event' 

`msg.event.ref_time.tv_sec' and msg.event.ref_time.tv_usec` is not the correct value either

(I already convert them from unix timestamp)

Any clue how to decode the alert timestamp ? so you know this library is helping me a lot to do my project and I need it so much

**Comment by kevin-hogan** [comment]
I realize it's been a couple of years since you wrote this, but I'm leaving this comment here in case someone else stumbles across this problem. The code as written assumes that the machine you're running Snort on will record each timestamp field as four bytes apiece, and that it's in big endian format. However, I found that in my environment (Snort 2.9.7.0 on a 64-bit docker container), I was getting each timestamp field as eight bytes and it was in little endian format. So I had to make some modifications to the unpacking code to get this working.
